sw/host/provisioning/ft/src/main.rs

```diff
@@ -73,6 +73,10 @@ pub struct ManufFtProvisioningDataInput {
     /// CA certificate to be used for verifying.
     #[arg(long)]
     pub ca_certificate: PathBuf,
+
+    // Cloud KMS Key ID of the key used for certificate endorsement.
+    #[arg(long, default_value = None)]
```

Comment:
it's a nice trick I did not know about, but we also need to enforce that one of the two is present in the command line, and the check in main() ensures this too. I cann add exclusivitiy requirement here and check main() to verify that one is presetn if you prefer, let me know.

Reply:
I _think_ you want something like this then: https://docs.rs/clap/latest/clap/_derive/_tutorial/chapter_3/index.html#argument-relations

Though you may also be able to make them both as "required = True", and add the conflicts_with, since according to here, conflicts_with takes precedence over required attributes: https://docs.rs/clap/latest/clap/struct.Arg.html#method.conflicts_with

Reply:
I can do this if you feel strongly about it, but I think this is an overkill:

we'd need a separate structure to put the two parameters in it, and we still need a run time `match`  o figure out which parameter is present.

Having `clap` dealing with it adds the benefit of bailing out early before any code in `main()` runs, but I am not sure if this is such a valuable benefit, let me know what you think.

Reply:
I think it's good practice to have both: adding it to clap has the benefit of documenting this conflict in the command-line description.